translations/monero-core_it.ts

```diff
@@ -1325,45 +1302,47 @@
     <message>
         <location filename="../components/TableDropdown.qml" line="182"/>
         <source>&lt;b&gt;Copy address to clipboard&lt;/b&gt;</source>
-        <translation type="unfinished"></translation>
+        <translation>&lt;b&gt;Copia indirizzo nella clipboard&lt;/b&gt;</translation>
     </message>
     <message>
         <location filename="../components/TableDropdown.qml" line="183"/>
+        <source>&lt;b&gt;Send to same destination&lt;/b&gt;</source>
```

Comment:
Looks like you have an error here (should only be 1 source and 1 translation).
The ones below ("**Send to this address**") are correct per [#685](https://github.com/monero-project/monero-core/pull/685/files)